apps/frontend/src/components/global/groups/GroupManagement.vue

```diff
@@ -113,14 +135,20 @@ import {Prop} from 'vue-property-decorator';
   }
 })
 export default class GroupManagement extends Vue {
-  @Prop({type: Boolean, default: false}) readonly allGroups!: boolean;
+  @Prop({type: Boolean, default: false}) readonly adminPanel!: boolean;
 
   editedGroup: IGroup | null = null;
   selectedGroupUsers: ISlimUser[] | null = [];
   dialogDelete = false;
   dialogDisplayUsers = false;
   search = '';
-  allGroupsHeaders: Object[] = [
+
+  groupsHeaders: Object[] = [
```

Comment:
'Object' is not a usable type, most things are 'Objects' in js-land.  make this a record, but we know what all the values are so you could also do a more literal type of object type where you either set up the type beforehand or just embed it in the array type

i..e

```
type GroupsHeader = {
  text: string,
  blah
}

groupsHeaders = GroupsHeader[]
```

or

```
groupsHeaders: {text: string, blah}[] =
```